### ISSUE TITLE
Crime shouldn't start until some amount of time after landing

### ISSUE THREAD

**Issue by ldicker83** [OutpostUniverse/OPHD#1229]
Related to #1228, criminal activity should probably not start until some amount of time _after_ the population has landed when the colonists have had some time to settle into their new home.

The idea behind this is that 1) colonists are going to be too busy adjusting to their new life to plan and commit crimes and 2) from a gameplay standpoint we want to give the player at least a few turns to establish themselves before crime is something they need to worry about.

This buffer period should be adjusted based on difficulty level. Default ('hard mode') should be 15 turns with no crime updates from the time that colonists are landed (just over 'one year')

**Comment by Inglonias** [comment]
Related issue that probably deserves its own issue number - Morale also starts dropping before colonists have landed.

**Comment by ldicker83** [comment]
Fair point. Thinking it should stay consistent while there's no population on the ground. Will open a new ticket for that. Thanks :D
